contrib/PKGBUILD

```diff
@@ -0,0 +1,35 @@
+# Maintainer: Bruno Pagani (a.k.a. ArchangeGabriel) <archange@archlinux.org>
+# Contributor: Mirco Tischler <mt-ml at gmx dot de>
+
+pkgname=fwupd
+pkgver=LATEST
```

Comment:
Hum… Do you know (I suppose so https://github.com/hughsie/fwupd/commit/b4208bb5776c90132a07c9d51f54c86ac007a734) why the parsing of meson.build is using such an expression? Why not just `head -n 2 ../../meson.build | tail -n 1 | cut -f2 -d \'`?